views/dashboard/about-page/components/FormView.tsx

```diff
@@ -0,0 +1,201 @@
+        {
+            name: '',
+            src: '',
+            title: '',
+            type: ''
+        }
+    ],
+    role: {
+        value: '',
+        error: ''
+    },
+    title: {
+        value: '',
+        error: ''
+    }
+}
+
+const validations = {
+    name: FormValidations.yolo,
+    description: FormValidations.yolo,
+    header: FormValidations.yolo,
+    subHeader: FormValidations.yolo,
+    img: FormValidations.yolo,
+    role: FormValidations.yolo,
+    title: FormValidations.yolo
+}
+
+const FormView = ({ FormData }: any) => {
+    const dispatch = useDispatch()
+
+    const { description, header, subHeader, img, role, title, name, id } =
+        FormData
+
+    const {
+        disableSubmit,
+        form,
+        formLoading,
+        handleChange,
+        handleFormSubmit,
+        handleNonFormEventChange,
+        handleImport,
+        isDataImported
+    } = useForm(initialState, validations, FormData)
+
+    const onFormSubmit = async () => {
+        try {
+            const cloudImg = await handleCloudUpload({
+                base64: form.img.value[0].base64,
+                context: 'aboutPage',
+                filename: form.img.value[0].name
+            })
+
+            const payload = {
+                ...(description !== form.description.value && {
+                    description: form.description.value
+                }),
+                ...(header !== form.header.value && {
+                    header: form.header.value
+                }),
+                ...(subHeader !== form.subHeader.value && {
+                    subHeader: form.subHeader.value
+                }),
+                ...(role !== form.role.value && { role: form.role.value }),
+                ...(title !== form.title.value && { title: form.title.value }),
+                ...(name !== form.name.value && { name: form.name.value }),
+                ...(img !== form.img.value && {
+                    src: cloudImg,
+                    type: form.img.value[0].type,
+                    name: form.img.value[0].name
+                })
+            }
+
+            const updatedAbout = await updateAboutPageData(id, payload)
+            dispatch(setAbout(updatedAbout))
+            enqueueSnackbar('Your About Me pagehas been updated.', {
+                variant: statuses.SUCCESS
+            })
+        } catch (error: any) {
+            enqueueSnackbar(
+                error.message === 'You about page update failed.'
+                    ? error.message
+                    : messages.TECHNICAL_DIFFICULTIES,
+                {
+                    variant: statuses.ERROR
+                }
+            )
+        }
+    }
+    const handleResetForm = () =>
+        handleImport({
+            description,
+            header,
+            subHeader,
+            img,
+            role,
+            title,
+            name
+        })
+
+    useEffect(() => {
+        if (id && !isDataImported) handleResetForm()
```

Comment:
This can't be right. Don't you mean to import the data here instead?

Reply:
Fuck do i know you had it in your code i pasted this bitch in and it worked . Am i gonna ask questions ? Hell tf no